organisers/python-novice-inflammation/data/01-intro.ipynb

```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Python Fundamentals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Questions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- What basic data types can I work with in Python?\n",
+    "- How can I create a new variable in Python?\n",
+    "- How do I use a function?\n",
+    "- Can I change the value associated with a variable after I create it?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Objectives"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Assign values to variables.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Content"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use Python as a calculator:\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In Python, we can assign a value to a variable, using the equals sign `=`. In layman’s terms, **a variable is a name for a value**. In Python, variable names:\n",
+    "- can include letters, digits, and underscores\n",
+    "- cannot start with a digit\n",
+    "- are case sensitive\n",
+    "\n",
+    "This means that, for example:\n",
+    "- `weight0` is a valid variable name, whereas `0weight` is not\n",
+    "- `weight` and `Weight` are different variables\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Record a patient weight:\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Check your understanding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Python allows you to assign multiple values to multiple variables in one line by separating the variables and values with commas. What does the following program print out?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "first, second = 'Grace', 'Hopper'\n",
+    "third, fourth = second, first\n",
+    "print(third, fourth)"
+   ]
+  },
```

Comment:
This first exercise in the first notebook contains some content that won't have been covere yet, like print() and strings. It's basic stuff so could probably still work fine but might be better later on in the episode. What do you think @patrick-austin?

Reply:
I committed a change so that it appears after the built-in functions section as then evrything will be covered by the time we get to it, feel free to revert the change if you disagree

Reply:
The second exercise also currently uses `print` and `type` in the solution. We could either move that later too, or remove the solution and just have the instructor state what the types are. On one hand I think it's nice to have regular exercises, but on the other I don't like the idea of just asserting what the answer is without showing it using the code. What do you think?

More generally tough I think it's a good spot and it's better to have less frequent assessments than it is to compromise the flow of the lesson by refering to things out of order since that's actively confusing to attendees.

Reply:
I agree with your final statement and as such we should move the second exercise you mentioned too.